runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -260,15 +382,28 @@ api.nvim_set_decoration_provider(namespace, {
   end,
 })
 
+--- Enable/disable/toggle inlay hints for a buffer
+---
+--- @param bufnr (integer) Buffer handle, or 0 for current
+--- @param enable_ (boolean|nil) true/false to enable/disable, nil to toggle
+function M.toggle(bufnr, enable_)
```

Comment:
Is the underscore a typo?

Reply:
It's from the original code; the reason is because `enable` is already defined in the module scope (`local function enable`)

Reply:
I see. I would be okay with it if this was only available/accessed in internal APIs, but its [name shows up in public documentation] (https://github.com/neovim/neovim/blob/54eeeba951ba468ab4a964b7baeedebdece6e39d/runtime/doc/lsp.txt#L1509), and so I think we should either rename the variable or change the hidden internal variables to handle the name conflict instead.

Reply:
That makes sense. Updated.